test/selenium/lib/view/log_in/view.js

```diff
@@ -0,0 +1,28 @@
+var View = require('../view');
+var LoginAccessors = require('./accessors');
+
+
+function LoginView() {
+  [].push.call(arguments, LoginAccessors);
+  View.apply(this, arguments);
+  
+  this.accessors.submitButton;
+}
+
+LoginView.prototype = Object.assign({
+
+  loginSuccess: function(password) {
```

Comment:
As we're entering both password and the password confirmation, I assume we're actually signing up. I would rename this function and its class. So we're clear on the fact that it's not just a regular log in.